src/stories/DetailCard.stories.js

```diff
@@ -1,6 +1,11 @@
 import React from "react";
 import { GamePositioned } from "../components/CollectionItem";
```

Comment:
is there a reason, that this one is not "export default"? I think you should stick to using the same method for every import if possible :)